<a href="https://colab.research.google.com/github/Mihail163Samara/29jule/blob/main/%D0%90%D1%82%D1%82%D0%B5%D1%81%D1%82%D0%B0%D1%86%D0%B8%D0%BE%D0%BD%D0%BD%D0%B0%D1%8F_%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%B0_%D0%BF%D0%BE_%D1%82%D0%B5%D0%BE%D1%80%D0%B8%D0%B8_%D0%B2%D0%B5%D1%80%D0%BE%D1%8F%D1%82%D0%BD%D0%BE%D1%81%D1%82%D0%B8_%D0%B8_%D0%BC%D0%B0%D1%82%D0%B5%D0%BC%D0%B0%D1%82%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B9_%D1%81%D1%82%D0%B0%D1%82%D0%B8%D1%81%D1%82%D0%B8%D0%BA%D0%B8_%D0%A3%D0%BC%D0%BD%D0%BE%D0%B2_%D0%9C_%D0%90.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Провести дисперсионный анализ для определения того, есть ли различия среднего роста среди взрослых футболистов, хоккеистов и штангистов. Даны значения роста в трех группах случайно выбранных спортсменов: Футболисты: 173, 175, 180, 178, 177, 185, 183, 182. Хоккеисты: 177, 179, 180, 188, 177, 172, 171, 184, 180. Штангисты: 172, 173, 169, 177, 166, 180, 178, 177, 172, 166, 170. Данная промежуточная аттестация оценивается по системе "зачет" / "не зачет". "Зачет" ставится, если Слушатель успешно выполнил задание. "Незачет" ставится, если Слушатель не выполнил задание. Критерии оценивания: 1 - Слушатель провел дисперсионный анализ для определения того, есть ли различия среднего роста среди взрослых футболистов, хоккеистов и штангистов.

Для проведения дисперсионного анализа (ANOVA) для определения того, есть ли различия среднего роста среди взрослых футболистов, хоккеистов и штангистов, выполним следующие шаги:

1. Соберем данные роста спортсменов по группам.

2. Проведем однофакторный дисперсионный анализ ANOVA.

3. Оценим результаты анализа.

In [3]:
import scipy.stats as stats

# Данные
football_players = [173, 175, 180, 178, 177, 185, 183, 182]
hockey_players = [177, 179, 180, 188, 177, 172, 171, 184, 180]
weightlifters = [172, 173, 169, 177, 166, 180, 178, 177, 172, 166, 170]

# Проведение однофакторного дисперсионного анализа
f_statistic, p_value = stats.f_oneway(football_players, hockey_players, weightlifters)

f_statistic, p_value


(5.500053450812596, 0.010482206918698694)

# **Результаты анализа**

По результатам однофакторного дисперсионного анализа ANOVA мы получили следующие значения:  

F-статистика: 5.500.

p-значение: 0.010.

**Интерпретация результатов**.

Значение F-статистики (5.500) указывает на наличие различий между группами. Чем больше значение F, тем больше различий между средними значениями групп.  


p-значение (0.010) указывает на вероятность того, что наблюдаемые различия в средних значениях могли возникнуть случайно. Обычно в статистике используют уровень значимости (α) 0.05.  


**Вывод**

Поскольку полученное p-значение (0.010) меньше уровня значимости 0.05, мы отвергаем нулевую гипотезу о том, что средние значения роста в трех группах (футболисты, хоккеисты и штангисты) равны. Это означает, что есть статистически значимые различия в среднем росте среди этих групп спортсменов.